From 599c7b05c432b1571a7105f1ebf6bbe30c36dbdf Mon Sep 17 00:00:00 2001
From: Kalev Lember <kalevlember@gmail.com>
Date: Thu, 28 Mar 2013 21:54:34 +0000
Subject: Adapt for gnome-calculator -> gcalctool desktop file rename

gnome-calculator.desktop was renamed back to gcalctool.desktop, at the
very last minute before the 3.8.0 release.

https://bugzilla.gnome.org/show_bug.cgi?id=696816
---
diff --git a/layout/gnome-applications.menu b/layout/gnome-applications.menu
index 84d13ed..503f9ca 100644
--- a/layout/gnome-applications.menu
+++ b/layout/gnome-applications.menu
@@ -38,7 +38,6 @@
         <Not><Filename>file-roller.desktop</Filename></Not>
         <Not><Filename>gnome-file-roller.desktop</Filename></Not>
         <Not><Filename>deja-dup-preferences.desktop</Filename></Not>
-        <Not><Filename>gnome-calculator.desktop</Filename></Not>
         <Not><Filename>gcalctool.desktop</Filename></Not>
         <Not><Filename>gucharmap.desktop</Filename></Not>
         <Not><Filename>gnome-gucharmap.desktop</Filename></Not>
@@ -251,7 +250,7 @@
     <Directory>X-GNOME-Utilities.directory</Directory>
     <Include>
       <Filename>file-roller.desktop</Filename>
-      <Filename>gnome-calculator.desktop</Filename>
+      <Filename>gcalctool.desktop</Filename>
       <Filename>gnome-font-viewer.desktop</Filename>
       <Filename>gucharmap.desktop</Filename>
       <Filename>seahorse.desktop</Filename>
@@ -287,7 +286,6 @@
         <Not><Filename>gnome-eog.desktop</Filename></Not>
         <Not><Filename>gnome-file-roller.desktop</Filename></Not>
         <Not><Filename>gnome-gucharmap.desktop</Filename></Not>
-        <Not><Filename>gcalctool.desktop</Filename></Not>
       </And>
     </Include>
   </Menu> <!-- End Other -->
--
cgit v0.9.1
